traffic_ops/traffic_ops_golang/region/regions_test.go

```diff
@@ -20,14 +20,13 @@ package region
  */
 
 import (
-	"testing"
-	"time"
-
 	"github.com/apache/trafficcontrol/lib/go-tc"
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/api"
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/test"
 	"github.com/jmoiron/sqlx"
 	sqlmock "gopkg.in/DATA-DOG/go-sqlmock.v1"
+	"testing"
+	"time"
```

Comment:
Import grouping should be:

1. Standard library
1. `golang.org/x/*`
1. External/vendored libraries (e.g. `sqlx`)
1. Project-local packages

Reply:
Done, thanks

Reply:
Nevermind definitely not resolved, gofmt wont let me follow that convention as-is, goland also formats it the way I pushed it

Reply:
I went with what gofmt did to it locally, is this okay to squash and proceed?